source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,159 @@
+=================
+Change a Document
+=================
+
+.. default-domain:: mongodb
+
+You can change documents in a MongoDB collection using two distinct
+operation types: :ref:`update <update-operation>` and :ref:`replace
+<replace-operation>`. Update operations change specified fields in one
+or more documents using the
+:java-docs:`updateOne() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateOne(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+or :java-docs:`updateMany() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#updateMany(org.bson.conversions.Bson,org.bson.conversions.Bson)>` 
+method. A replace operation substitutes one document with a new
+specified document using the
+:java-docs:`replaceOne() </apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#replaceOne(org.bson.conversions.Bson,TDocument)>` 
+method.
+    
+In the following examples, a paint store sells eight different
+colors of paint. The store had their annual online sale. Their
```

Comment:
I'm not sure this detail about their annual online sale makes sense here. This is the first time we're showing the data. It's also not clear whether any inventory was sold in the annual online sale. Instead, consider just mentioning that the collection represents their current inventory in the ``paint_inventory`` collection.